test/c/eacsl/not-yet-supported/abrupt_termination/returns.t

```diff
@@ -1,2 +1,12 @@
   $ owi c --e-acsl ./returns.c
-  All OK
+  [e-acsl] returns.c:8: User Error: 
+    Term '\at(x,__invalid_label)' was used before being translated.
+    This usually happens when using a label defined after the place
+    where the Term should be translated
+  [kernel] Plug-in e-acsl aborted: invalid user input.
```

Comment:
Doesn't this mean that the mistake is ours here ? The error message looks suspicious to me

Reply:
It's the example on the E-ACSL reference and I guess the support for `returns` is incomplete (in fact the implementation reference marks it as non-supported)

Reply:
OK, maybe it should be good to open an issue on e-acsl to ask about changing this to a warning and to ignore `returns` ?

Reply:
Ok